sharding-core/src/main/java/in/cleartax/dropwizard/sharding/transactions/TransactionRunner.java

```diff
@@ -53,6 +56,8 @@ public T start(boolean reUseSession, UnitOfWork unitOfWork) throws Throwable {
             aspect.onFinish();
             DelegatingTenantResolver.getInstance().setDelegate(null);
         }
+        log.info("[DATABASE] transaction={} error={} operation={} time-elapsed={}",
```

Comment:
What will be the suitable log level? `TRACE`?

Reply:
It could be part of setting.